core/pom.xml

```diff
@@ -114,6 +110,10 @@
                     <groupId>org.slf4j</groupId>
                     <artifactId>slf4j-simple</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
it think we should also exclude compile dependency to com.google.code.findbugs:jsr305:jar:3.0.2:compile?

Reply:
good point, exclusion added in https://github.com/apache/sling-org-apache-sling-testing-osgi-mock/pull/18/commits/2912b732457c5b0546668c618774d5cd9cfbbdeb.